docs/core/deploying/deployment-experiences.md

```diff
@@ -0,0 +1,52 @@
+---
+title: .NET Core Deployment Experiences
+description: .NET Core Deployment Experiences
+author: jamshedd
+ms.author: jamshedd
+ms.date: 09/06/2019
+ms.custom: 
+---
+# .NET Core Deployment Experiences
+
+## Full Installers
+.NET Core major and minor releases, and servicing updates always ship as the full product installers, that is the installer carries the entire product even if only a subset of the files is updated.
+
+## Side by Side installs versus in-place updates
```

Comment:
It is probably worth noting in this section for those who have been regularly broken by in place updates, that SCD is the way they can ensure they are not broken by in-place updates

Reply:
Ack.